packages/opds-web-client/src/components/LaneMoreLink.tsx

```diff
@@ -1,49 +1,27 @@
 import * as React from "react";
+import "../stylesheets/lane_more_link.scss";
 import CatalogLink from "./CatalogLink";
 
 export default class LaneMoreLink extends React.Component<any, any> {
   render() {
-    let cellStyle = Object.assign({
-      display: "table-cell",
-      verticalAlign: "middle",
-      backgroundColor: "#eee",
-      textAlign: "center",
-      padding: "10px",
-      color: "#888",
-      fontWeight: "bold",
-      width: "150px",
-      height: "200px"
-    }, this.computeFontStyle(), this.props.style);
-
-    let moreLinkStyle = {
-      display: "block",
-      height: "250px",
-      whiteSpace: "normal", // overrides laneBookStyle
-      textDecoration: "none"
-    };
+    let fontSize = this.computeFontSize();
 
     return (
       <CatalogLink
-        className="moreLink"
-        style={moreLinkStyle}
+        className="more-link"
         collectionUrl={this.props.lane.url}>
-        <div style={cellStyle}>
+        <div style={{ fontSize: fontSize }}>
```

Comment:
It's the link at the end of a lane when you scroll all the way to the right that's the size of a book cover and says "More Young Adult Fiction" or whatever the lane name is.